README.md

```diff
@@ -207,6 +207,20 @@ You can run checkstyle using:
 The checkstyle warnings will be found in `reports/checkstyle/reports/main.html` and `reports/checkstyle/reports/test.html` files in the
 subproject build directories. They are also printed to the console. The build will fail if Checkstyle fails.
 
+As of present, the auto-formatting configuration is work in progress. Auto-formatting is automatically invoked for the modules listed below when the 'checkstyleMain' or 'checkstyleTest' task is run.
+
+- (No modules specified yet)
+
+You can also run auto-formatting independently for a single module listed above, like:
+
+    ./gradlew :core:spotlessApply   # auto-format *.java files in core module, without running checkstyleMain or checkstyleTest.
+
+If you are using an IDE, you can use a plugin that provides real-time automatic formatting. For detailed information, refer to the following links:
+
+- [Eclipse](https://checkstyle.org/eclipse-cs)
+- [Intellij](https://plugins.jetbrains.com/plugin/1065-checkstyle-idea)
+- [Vscode](https://marketplace.visualstudio.com/items?itemName=shengchen.vscode-checkstyle)
+
```

Comment:
I would remove this until we have it setup for Streams.

Reply:
I thought it would be better to update the list every time the streams' packages are configured to auto-formatting - and since I am ready for this task, it will not need much time.

Moreover, These descriptions are applied to the whole project, isn't it?

Reply:
Yes, but before at least one module does not apply automatic formatting, I would remove this description because until then it is useless.

Reply:
Okay. I will move this section into the other subissue. :+1: